frontend/src/static/lib/Inter/inter.css

```
@@ -0,0 +1,65 @@
+@font-face {
+	font-family: 'Inter';
```

Stylelint (17.9.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)